client/.eslintrc.js

```diff
@@ -23,6 +23,11 @@ module.exports = {
   env: {
     browser: true,
     es6: true,
+    commonjs: true
+  },
+  globals: {
+    // Webpack plugin adds `process` global
+    process: false
```

Comment:
I understood the "define" as in "tell ESLint about the existence of this variable", not as in "define a new global variable called `process`. And yeah, setting the value `false` to my understanding means that all attempts to change the `process`, e.g. `process.env.NODE_ENV = "development"` are forbidden.